src/index.ts

```diff
@@ -97,7 +104,7 @@ async function scanDeps(config: ScanDepsConfig): Promise<ScanDepsResult> {
   // Example: "@ms/my-package" --> "@ms/my-package"
   const packageRegExp: RegExp = /^((@[a-z\-0-9!_]+\/)?[a-z\-0-9!_]+)\/?/;
 
-  const requireMatches: Set<string> = new Set<string>();
+  const requireMatches: DepMapWithFileAndLine = {};
```

Comment:
IMO, we should stick to string[] or Set type to record name, and use a brand new Map or object type to record file & lineNumber info.

e.g. 

```
detectedDependencies = ['react'];

detectedDependencyNameToInfo = {
  react: [
    { filePath: 'xxx/src/App.tsx', lineNumber: 1 },
    { filePath: 'xxx/src/components/foo.tsx', lineNumber: 2 },
  ],
}
```

Reply:
Done